src/aktualizr_primary/main.cc

```diff
@@ -131,11 +131,9 @@ int main(int argc, char *argv[]) {
       try {
         Primary::initSecondaries(aktualizr, config.uptane.secondary_config_file);
       } catch (const std::exception &e) {
-        LOG_ERROR << "Secondary initialization failed: " << e.what();
-        if (!aktualizr.IsRegistered()) {
-          LOG_ERROR << "Cannot provision without all secondaries present, exiting...";
-          return EXIT_FAILURE;
-        }
+        LOG_ERROR << e.what();
```

Comment:
Just minor comment. Perhaps, it makes sense to add some message before `e.what` or combine it with the next LOG_ERROR.